### PR TITLE
Corrected hyperlink generation for headers.

### PR DESCRIPTION
**Pull request by ESapenaVentura**
Updated `src/human_readable_json.py` to generate the header hyperlinks correctly

ebi-ait/hca-ebi-wrangler-central#846

### CODE REVIEW

File: tests/python/test_human_readable_json.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from unittest import TestCase
```
Comment (ESapenaVentura) [inline tests/python/test_human_readable_json.py:1]: When is this test triggered?

**Review by amnonkhen — APPROVED** [review]

File: tests/python/test_human_readable_json.py (hunk 1 above)
Comment (amnonkhen) [reply tests/python/test_human_readable_json.py:1]: Currently it is not connected to any automated process.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	PYTHONPATH=$(SRC) $(PYTHON) -munittest discover --start $(TESTS)/python
+
+$(TESTS)/package.json:
+
+$(TESTS)/node_modules/: $(TESTS)/package.json
+	cd $(TESTS) ; npm install ; touch node_modules/
+
+.PHONE: test-node
+test-node: $(TESTS)/schema.test.js $(TESTS)/node_modules
+	cd $(TESTS) ; npm test
```
Comment (amnonkhen) [inline Makefile:19]:
```suggestion
	cd $(TESTS) ; npm test
	
```

File: tests/python/test_human_readable_json.py (hunk 1 above)
Comment (ESapenaVentura) [reply tests/python/test_human_readable_json.py:1]: Instead of having this make file, wouldn't it make sense to integrate it with the github actions?

Comment (MightyAx) [reply tests/python/test_human_readable_json.py:1]: added to GitHub Actions, as requested.

**Review by MightyAx — APPROVED** [review]

**Comment by ESapenaVentura** [comment]
LGTM - Let's merge it!

**Pull request merged**